File: easy/217-Contains_Duplicate.java
public boolean containsDuplicate(int[] nums) {
	HashSet<Integer> set = new HashSet<>();
	for(int a = 0; a < nums.length; a++) {
		if(set.contains(nums[a]))
			return true;
		set.add(nums[a]);
	}
	return false;
}

File: easy/217-Contains_Duplicate.cpp
bool containsDuplicate(vector<int>& nums) {
	unordered_set<int> set;
	
	for(int a = 0; a < nums.size(); a++) {
		if(set.count(nums[a]) == 1)
			return true;
		set.insert(nums[a]);
	}
	return false;
}